css/style.css

```diff
@@ -525,3 +525,247 @@ body {
   50% {transform: scale(0.67);}
   100% {transform: scale(1);}
 }
+
+.google-icons {
+  position: absolute;
+  width: 100px;
+  height: 100px;
+  top: 479px;
+  background-repeat: no-repeat;
+  transform: translate(-50%, 100%);
+  background-size: 100%;
+  opacity: 0;
+}
+
+.impressions-text {
+  line-height: 1.2;
+  text-align: center;
+  font-size: 2em;
+  font-weight: bold;
+  color: #343536;
+  position: absolute;
+  top: 20%;
+  padding: 10px;
+  display: grid;
+}
+
+.impressions {
+  width: 414px;
+  height: 736px;
+  position: relative;
+}
+
+.impressions-background {
+  height: 100%;
+  width: auto;
+  background-image: url(../image/desktopcalendar.png);
+  background-repeat: no-repeat;
+  background-position: 50% 50%;
+  background-size: auto;
+  filter: blur(1px);
+  filter: opacity(0.4);
+}
+
+.click-icon {
+  width: 100px;
+  height: 100px;
+  position: relative;
+  background-image: url(../image/click.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  transform: translate(50% 50%);
+  top: 30px;
+  left: 35%;
+  background-position: center center;
+  background-size: 100%;
+}
+
+h1 {
+  font-weight: bolder;
+  text-align: center;
+  padding-top: 40%;
+  font-size: 4em;
+  color: #fbbc04;
+}
+
+p {
+  font-weight: bold;
+  text-align: center;
+  font-size: 3em;
+  line-height: 1;
+  padding-bottom: 10%;
+  padding-left: 10%;
+  padding-right: 10%;
+}
+
+.icons {
+  position: absolute;
+  width: 111px;
+  height: 111px;
+  left: 34px;
+  opacity: 1;
+}
+
+.group1 {
+  top: 126px;
+}
+
+.group2 {
+top: 305px;
+}
+
+.group3 {
+top: 484px;
+}
+
+.content {
+  position: absolute;
+  left: 190px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  font-style: normal;
+  font-family: 'Open Sans', sans-serif;
```

Comment:
Can you change font to **Google Sans** for consistency?

Reply:
done.